### PR TITLE
Add meal category to recipe cards ts

### PR DESCRIPTION
**Pull request by tauri-st**
- Add meal category to recipe card JSX in JS
- I created an empty variable to accept the first meal category found and used that as a place holder
- I also adjusted the padding and font color of the meal category on the cards
- and I adjusted the size of the meal category in the modal
- and the size of the modal header
- I also changed the hierarchy of the headers in the modal so that the meal category is h2 instead of h3 and instructions + ingredients are h3, the hierarchy being in order is important for screen readers who can index items on a page + I believe HTML validators

### CODE REVIEW

File: docs/js/main.js [hunk 1]
```diff
@@ -97,10 +97,22 @@ function showRecipes(recipes) {
         const recipeID = recipes[recipe].id;
         const title = recipes[recipe].name;
         const thumbnail = recipes[recipe].thumbnail_url;
+        // get meal category from first meal tag
+        const tagsArray = recipes[recipe].tags
+        let foundMealCategory = "";
+        for (const recipe in tagsArray) {
+            if (tagsArray[recipe].root_tag_type == "meal") {
+                let mealCategory = tagsArray[recipe].display_name;
+                //modalCategory.innerHTML = mealCategory;
```
Comment (terrifricker) [inline docs/js/main.js]:
```suggestion
```

**Review by terrifricker — CHANGES_REQUESTED** [review]
Such a great job, @tauri-st!

Thanks for grabbing those other items like fixing the headings and making the modal header better.

I tried to use the "suggest a change" right in the code to we could experience this GitHub use.  The only change was to remove a commented out line of code.  I think you can accept the change right in that dialog box.

**Review by chonger878 — APPROVED** [review]
Everything looks great! Awesome pair programming!

**Review by tauri-st — COMMENTED** [review]
Ok, I accepted the change thru Github! I think you may need to review @terrifricker before it can be merged

**Review by terrifricker — APPROVED** [review]

**Pull request merged**